pynestml/frontend/pynestml_frontend.py

```diff
@@ -107,14 +108,18 @@ def builder_from_target_name(target_name: str, options: Optional[Mapping[str, An
 
     if target_name.upper() == "NEST":
         from pynestml.codegeneration.nest_builder import NESTBuilder
-        return NESTBuilder(options)
+        if options.pop("redirect_build_output", None):
+            nest_builder = NESTBuilder(options)
+            nest_builder.redirect_build_output(FrontendConfiguration.get_target_path())
```

Comment:
I think it would be simpler to have the `redirect_build_output` option as part of `codegen_opts`. The `if` condition would be avoided and the `options` would be parsed in the `Builder` class. See the option `nest_path` for example.